trimesh/io/wavefront.py

```diff
@@ -193,23 +202,29 @@ def append_mesh():
                 loaded['metadata']['face_groups'] = face_groups
 
             if len(current['usemtl']) > 0:
-                current_mtllibs = [mtllibs[k] for k in current['usemtl']
-                                   if k in mtllibs]
-                try:
-                    vertex_colors = _get_vertex_colors(
-                        uv=loaded['metadata']['vertex_texture'],
-                        mtllibs=current_mtllibs,
-                        resolver=resolver)
-                    loaded['vertex_colors'] = vertex_colors
-                    loaded['metadata']['mtllibs'] = current_mtllibs
-                except BaseException:
-                    log.error('failed to load texture!', exc_info=True)
+                texture = np.array(current['vt'], dtype=np.float64)
+                vertex_colors = np.zeros((0, 4), dtype=np.uint8)
+                for usemtl, findices in usemtl_to_findices.items():
+                    uv = texture[findices]
+                    try:
+                        vertex_colors_i = _get_vertex_colors(
+                            uv=uv, mtllib=mtllibs[usemtl], resolver=resolver)
+                    except BaseException:
+                        log.error('failed to load texture: {}'.format(usemtl),
+                                  exc_info=True)
+                        vertex_colors_i = None
+                    if vertex_colors_i is None:
+                        vertex_colors_i = np.full(
+                            (len(uv), 4), (102, 102, 102, 255), dtype=np.uint8)
```

Comment:
It seems this gray color is the default. Is this correct?

Reply:
Yeah, or since visual is already imported if you prefer you can use `visual.DEFAULT_COLOR`